oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -27,45 +27,34 @@ use std::{
     collections::HashMap,
     convert::Infallible,
     net::{Ipv6Addr, SocketAddr},
-    path::PathBuf,
     process::Command,
     sync::{Arc, Mutex},
 };
 
+/// Returns the path to the Wasm file produced by compiling the provided `Cargo.toml` file.
+fn build_wasm_module_path(metadata: &cargo_metadata::Metadata) -> String {
+    let package_name = &metadata.root_package().unwrap().name;
+    // Keep this in sync with `/runner/src/main.rs`.
+    format!("{}/bin/{}.wasm", metadata.workspace_root, package_name)
+}
+
 // TODO(#1965): Move this and the similar function in `oak/sdk` to a common crate.
 /// Uses cargo to compile a Rust manifest to Wasm bytes.
-pub fn compile_rust_wasm(
-    manifest_path: &str,
-    module_wasm_file_name: &str,
-) -> anyhow::Result<Vec<u8>> {
-    let mut module_path = PathBuf::from(manifest_path);
-    module_path.pop();
-    module_path.push("bin");
-
+pub fn compile_rust_wasm(manifest_path: &str) -> anyhow::Result<Vec<u8>> {
```

Comment:
Why is it copied from `oak_tests` here? To remove the dependency?

Reply:
To make sure that it is using the same compilation logic, and caches are reused.